src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -1464,40 +1465,40 @@ void LedgerConsensusImp::takeInitialPosition (
     initialSet = initialSet->snapShot (false);
 
     // Tell the ledger master not to acquire the ledger we're probably building
-    ledgerMaster_.setBuildingLedger (mPreviousLedger->info().seq + 1);
+    _ledgerMaster.setBuildingLedger (_previousLedger->info().seq + 1);
 
     auto txSet = initialSet->getHash ().as_uint256();
-    JLOG (j_.info()) << "initial position " << txSet;
+    JLOG (_j.info()) << "initial position " << txSet;
     mapCompleteInternal (txSet, initialSet, false);
 
-    mOurPosition = std::make_shared<LedgerProposal> (
-        initialLedger->info().parentHash, txSet, mCloseTime);
+    _ourPosition = std::make_shared<LedgerProposal> (
+        initialLedger->info().parentHash, txSet, _closeTime);
 
-    for (auto& it : mDisputes)
+    for (auto& it : _disputes)
     {
         it.second->setOurVote (initialLedger->txExists (it.first));
     }
 
     // if any peers have taken a contrary position, process disputes
     hash_set<uint256> found;
 
-    for (auto& it : mPeerPositions)
+    for (auto& it : _peerPositions)
     {
         uint256 set = it.second->getCurrentHash ();
```

Comment:
Micro-optimization: it _should_ be safe to change this to `auto const& set = it.second->getCurrentHash()`.